src/main/java/com/epam/aidial/core/controller/PublicationController.java

```diff
@@ -93,10 +94,10 @@ public Future<?> deletePublication() {
         context.getRequest()
                 .body()
                 .compose(body -> {
-                    Publication publication = ProxyUtil.convertToObject(body, Publication.class);
-                    ResourceDescription resource = decodePublication(publication.getUrl(), false);
+                    String url = ProxyUtil.convertToObject(body, ResourceLink.class).url();
```

Comment:
Is permission check not required any longer?

Reply:
url is no longer required